diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -988,6 +988,38 @@ fun ASTNode.visit(visitor: AstNodeVisitor<Unit>) {
 fun PsiElement.isLongStringTemplateEntry(): Boolean =
     node.elementType == LONG_STRING_TEMPLATE_ENTRY
 
+/**
+ * Checks that identifier is correct backing field
+ *
+ * @return true if the function is correct backing field
+ */
+internal fun ASTNode.isCorrectBackingField(variableName: ASTNode): Boolean {
```

Comment:
should we remove this method from previous place?

Reply:
Yes, done